parlai/core/params.py

```diff
@@ -118,6 +118,10 @@ def add_model_args(self):
             agent = get_agent_module(model)
             if hasattr(agent, 'add_cmdline_args'):
                 agent.add_cmdline_args(self)
+            if hasattr(agent, 'dictionary_class'):
+                self.parser.add_argument(
+                    '-dict_class', default=agent.dictionary_class(), type=str,
```

Comment:
two hyphens at start, hyphen instead of underscore like we talked about before: `--dict-class`